tests/test_websocket_handler.c

```diff
@@ -24,6 +24,8 @@
 #    pragma warning(disable : 4204) /* non-constant aggregate initializer */
 #endif
 
+static const size_t s_default_initial_window_size = 256;
```

Comment:
```
/* Use small window so that we can observe it opening in tests.
 * Channel may wait until the window is small before issuing the increment command. */
```